auto_nag/scripts/inactive_ni_pending.py

```diff
@@ -193,6 +194,12 @@ def _needinfo_triage_owner_flag(bug):
 
     @staticmethod
     def _request_from_triage_owner(bug):
+        if (
+            len(bug["inactive_ni"]) == 1
+            and bug["inactive_ni"][0]["setter"] == bug["triage_owner"]
+        ):
+            return "could you find another way to get the information or close the bug as `INCOMPLETE` if it is not actionable?"
```

Comment:
Could we mention the reasons why we forward in this case too?

Reply:
Don in 6b2f0e571b9c104dee6f6a97b6a302269c35c5f9.

@marco-c Do you think it feels contradictory to mention that the bug is important and at the same time ask to close the bug, example: ":owner, since the bug has high priority, high severity and recent activity, could you find another way to get the information or close the bug as `INCOMPLETE` if it is not actionable?"

I feel that not mentioning the reasons here will be less confusing.

Reply:
Let's not mention the reasons then